Classes/common/HTTP/CDTHTTPInterceptorContext.m

```diff
@@ -16,6 +16,12 @@
 
 #import "CDTHTTPInterceptorContext.h"
 
+@interface CDTHTTPInterceptorContext ()
+
+@property (nullable, readwrite, nonatomic, strong) NSURLResponse *response;
```

Comment:
actually, that isn't the file I thought it was, we can just access the iVar, but this way I can just have the compiler generate the code for me.